subprojects/language-java/language-java.gradle

```diff
@@ -7,14 +7,24 @@ dependencies {
     compile project(":platformJvm")
     compile project(":languageJvm")
 
+    // // INCAP!
+    compile 'com.github.gradle:incap:-SNAPSHOT'
+
     // TODO - get rid of this cycle
     integTestRuntime project(':plugins')
 }
 
 // These public packages have classes that are tangled with the corresponding internal package.
 useClassycle(exclude: ["org/gradle/api/tasks/compile/**", "org/gradle/external/javadoc/**", ])
 
-strictCompile()
+// jitpack seems to put incap test projects on the classpath,
+// which results in "no annotation processor claimed these" errors.
+// To get things compiling.
+compileJava.options.annotationProcessorPath = project.files()
```

Comment:
There should be a dependency on the _client_ of incap only, which shouldn't bring any other dependency on classpath. I understand the hack, but this must not make its way to the final PR.